scene/resources/visual_shader.cpp

```diff
@@ -1129,6 +1141,36 @@ bool VisualShader::is_port_types_compatible(int p_a, int p_b) const {
 	return MAX(0, p_a - (int)VisualShaderNode::PORT_TYPE_BOOLEAN) == (MAX(0, p_b - (int)VisualShaderNode::PORT_TYPE_BOOLEAN));
 }
 
+void VisualShader::attach_node_to_frame(Type p_type, int p_node, int p_frame) {
+	ERR_FAIL_INDEX(p_type, TYPE_MAX);
+	Graph *g = &graph[p_type];
+
+	ERR_FAIL_COND(!g->nodes.has(p_node));
+	ERR_FAIL_COND(p_frame < 0);
```

Comment:
```suggestion
	ERR_FAIL_COND(p_frame < 0);
	Graph *g = &graph[p_type];

	ERR_FAIL_COND(!g->nodes.has(p_node));
```
I'd say to not delay after lookup for this check